packages/p2p/src/components/add-payment-method-form/add-payment-method-form.tsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
-import { observer } from 'mobx-react-lite';
-import { Field, Form } from 'formik';
+import { Field, Form, FormikValues } from 'formik';
 import { Button, Icon, Input, Loading, Text } from '@deriv/components';
 import { isDesktop, isMobile } from '@deriv/shared';
-import { Localize } from 'Components/i18next';
+import { observer } from '@deriv/stores';
 import { useStores } from 'Stores';
+import { TPaymentMethodFieldMapProps } from 'Types';
+import { Localize } from 'Components/i18next';
 import ModalForm from 'Components/modal-manager/modal-form';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
```

Comment:
**non-blocking:** re-arrange the import orders
... from `Components/...`
... from `Stores`
... from `Types`